packages/adapters/ethers/src/client.ts

```diff
@@ -1112,7 +1109,19 @@ export class EthersAdapter {
 
   private async syncAuthConnector(projectId: string, bypassWindowCheck = false) {
     if (bypassWindowCheck || typeof window !== 'undefined') {
-      this.authProvider = W3mFrameProviderSingleton.getInstance(projectId)
+      this.authProvider = W3mFrameProviderSingleton.getInstance({
+        projectId,
+        onTimeout: () => {
+          AlertController.open(ErrorUtil.ALERT_ERRORS.INVALID_APP_CONFIGURATION, 'error')
+          // eslint-disable-next-line no-console
+          console.error(
+            ErrorUtil.ALERT_ERRORS.ORIGIN_NOT_WHITELISTED_SOCIALS.replace(
+              '{{origin}}',
+              window.origin
+            )
+          )
+        }
```

Comment:
I really wanted to add this logic in [W3mFrameProvider](https://github.com/reown-com/appkit/blob/main/packages/wallet/src/W3mFrameProvider.ts), but then i can't use `AlertController`.

Because i'm running to an issue cyclic dependency issue where `@reown/appkit-wallet` is used in `@reown/appkit-core`, so we can't do the same thing the other way around with `@reown/appkit-core`. 

You can try adding `@reown/appkit-core` [here](https://github.com/reown-com/appkit/blob/main/packages/wallet/package.json) and you'll get the error.